www/front_src/src/Resources/index.tsx

```diff
@@ -172,34 +183,22 @@ const Resources = (): JSX.Element => {
 
   React.useEffect(() => {
     initAutorefreshAndLoad();
-  }, [
-    sortf,
-    sorto,
-    page,
-    limit,
-    currentSearch,
-    states,
-    statuses,
-    resourceTypes,
-    hostGroups,
-    serviceGroups,
-  ]);
+  }, [sortf, sorto, page, limit, currentSearch]);
+
+  const firstUpdate = React.useRef(true);
+  React.useEffect(() => {
+    if (firstUpdate.current) {
+      // to avoid duplicate rendering on mount
+      firstUpdate.current = false;
```

Comment:
What causes the double render?

Reply:
There are 2 useEffect which induces double render : 

- https://github.com/centreon/centreon/pull/8586/files#diff-ca62ddbe4609c0067343bddbddf3f09aR184
- https://github.com/centreon/centreon/pull/8586/files#diff-ca62ddbe4609c0067343bddbddf3f09aR189

Reply:
the other solution is to use `setCurrentSearch(nextSearch);` in each function who handle filter changes, but I think it's too verbose